commit 3f34fb893070a66995b407b4c6cf71c3598eefd8
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Tue Dec 1 21:51:00 2015 +0100

    Make Qt5OpenGL optional, currently it isn't even in use

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e0187bb..3a241df 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,7 +16,8 @@
 set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH})
 set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules" ${CMAKE_MODULE_PATH})
 
-find_package(Qt5 5.2 REQUIRED COMPONENTS Gui Qml Quick Xml Sql Svg OpenGL PrintSupport Multimedia Test)
+find_package(Qt5 ${QT_MIN_VERSION} REQUIRED COMPONENTS Gui Qml Quick Xml Sql Svg PrintSupport Multimedia)
+find_package(Qt5OpenGL ${QT_MIN_VERSION} COMPONENTS)
 include(KDEInstallDirs)
 include(KDECompilerSettings)
 include(KDECMakeSettings)
